### PR TITLE
Check Cloud & SUSE-Cloud-5-{Pool,Updates} repos

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

**Comment by jsuchome** [comment]
+1

File: lib/suse-cloud-upgrade-4-to-5-pre [hunk 1]
```diff
@@ -178,17 +178,15 @@ fi
...
 # We need to ensure that Cloud5 main repository and update repos are in place
 
-# FIXME repos that do not exist yet
-REPOS_SKIP_CHECKS+=" Cloud SUSE-Cloud-5-Pool SUSE-Cloud-5-Updates"
-
 check_media_content \
     Cloud \
     /srv/tftpboot/suse-11.3/repos/Cloud \
-    1558be86e7354d31e71e7c8c2574031a
+    #1558be86e7354d31e71e7c8c2574031a
```
Comment (jsuchome) [inline lib/suse-cloud-upgrade-4-to-5-pre]: So, we don't need to skip it, but we're not checking md5 sum... isn't it just a diffeernt kind of skip?

Comment (vuntz) [inline lib/suse-cloud-upgrade-4-to-5-pre]: Now we check that the GPG key to sign the content file is one we know about.

**Comment by jsuchome** [comment]
It's true this is better. +1

**Comment by aplanas** [comment]
Partially tested.  I created fake repos and the script detect the wrong signature properly.
+1

**Pull request merged**